vip-portal/src/main/resources/vm/wrapper.vm

```diff
@@ -19,6 +19,23 @@ function error {
   echo [ ERROR - $D ] $* >&2
 }
 
+function function_singularity {
+    cat > singularity << 'EOF'
+          #!/bin/bash
+          /cvmfs/dirac.egi.eu/container/apptainer/bin/apptainer "$@"
+EOF
+}
+
+# Call the function to create the singularity script and make it executable
+function_singularity
```

Comment:
We have the choice to always add singularity/apptainer (like you did), or to do it only when the boutiques app uses singularity. (like it's done for udocker).
@camarasu what do you prefer ?

Reply:
done